src/main/io/gps/gps.h

```diff
@@ -131,6 +133,12 @@ typedef struct gpsSolutionData_s {
     uint8_t numSat;
 } gpsSolutionData_t;
 
+typedef struct ubxMonVer_s {
+    char swVersion[30];
+    char hwVersion[10];
+    char extension[10 * 30];
+} ubxMonVer_t;
```

Comment:
btw do we need this huge struct? 340 bytes to save something on the FC seems excessive. Can't we just read related settings from the GPS unit if we need them?

Reply:
We can extract only what we need like https://github.com/betaflight/betaflight/pull/12764/files#diff-15b828e75456a052109384cb9af7da6fb6001ebdf63c417647471ca2201cdee8R1644-R1657

Reply:
This struct shall be used only in `ubxPayload_t`, it's size is larger anyway

Reply:
Sorry, only in gps receive message `_buffer` (instead of defining it inplace)

Reply:
@ledvinap Currently we do not make any use of the `extension` data. We only use the `hwVersion` value.  Please advise if we need to define the `extension` variable here, or not, from the perspective of uart buffer assignment.